DuckDuckGo/TabViewControllerBrowsingMenuExtension.swift

```diff
@@ -28,25 +28,32 @@ import PrivacyDashboard
 
 extension TabViewController {
 
-    func buildBrowsingMenuHeaderContent() -> [BrowsingMenuEntry] {
+    private var shouldShowAIChatInMenuHeader: Bool {
+        let settings = AIChatSettings(privacyConfigurationManager: ContentBlocking.shared.privacyConfigurationManager,
+                                      internalUserDecider: AppDependencyProvider.shared.internalUserDecider)
+        return settings.isAIChatBrowsingToolbarShortcutFeatureEnabled
+    }
+
+    private var shouldShowPrintButtonInBrowsingMenu: Bool { shouldShowAIChatInMenuHeader }
```

Comment:
Silly question: is this intended? This'll mean the Print button is hidden when the AI chat option is disabled 😅

Reply:
It's intended yes. The name clearly needs work then haha. Basically, if AI chat is on, we MOVE the print button from the Browswing menu header to the browsing menu list. So if this flag is ON, it means the print button should be in the list instead of the header.

I'll rename this from `shouldShowPrintButtonInBrowsingMenu` to `shouldShowPrintButtonInBrowsingMenuList`, wdyt?

Reply:
Ohh, I get it now - that naming change sounds fine to me, yeah. Thanks for clarifying!